deploy/pvw/simulations/templates/default/run.tcl

```diff
@@ -718,6 +718,8 @@ puts $fileID "0.00000001        ! Molecular Diffusivity"
 puts $fileID "0.25d0        ! fraction of Dx/Vx for numerical stability"
 puts $fileID "0             ! Number of indicators provided. If this value is great than 0 an indicator file must be included"
 puts $fileID "\"\"            ! Name of the indictor file to use set to '' if not using an indicator file"
+puts $fileID "True          ! use velfile"
+# puts $fileID "\"\"          ! Boolname not allowed in Sept 10 2021 version"
```

Comment:
Is this comment needed? If so can we make it more explicit, It's not clear to me what this means